lib/localparticipant.js

```diff
@@ -549,7 +549,10 @@ class LocalParticipant extends Participant {
  * A {@link LocalTrack} was added by the {@link LocalParticipant}.
  * @param {LocalTrack} track - The {@link LocalTrack} that was added
  * @event LocalParticipant#trackAdded
- * @deprecated
+ * @deprecated If you are using this event to monitor {@link LocalTrack}s that
+ *   are published to the {@link Room}, alternatively you can access the .tracks
+ *   collection which contains the newly added {@link LocalTrack}s right after
+ *   LocalParticipant#publishTrack(s) is called
```

Comment:
Nice, please place the "LocalParticipant#publishTrack" in an `{@link ...}`, though.